src/patternfly/components/Table/table-td.hbs

```diff
@@ -6,6 +6,7 @@
     {{else if table-td--action}} class="pf-c-table__action"
     {{else if table-td--modifier}} class="{{table-td--modifier}}"
     {{/if}}
+    role="cell"
```

Comment:
Within `.pf-c-table__expandable-row.pf-m-expanded`, if empty cells are present, VO announces them where it should ignore them. Omitting `role="cell"` attr on them hides. 

```
{{#unless table-td--IsEmpty}}
  role="cell"
{{/unless}}
```

Reply:
Oh yes good point!

Reply:
updated

Reply:
![Screen Shot 2020-01-21 at 1 38 21 PM](https://user-images.githubusercontent.com/5385435/72832656-50933c00-3c53-11ea-9c92-ed78d07fd82e.png)

https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Table/table-td.hbs#L9
```
{{#unless table-td--IsEmpty}}
  role="cell"
{{/unless}}
```

then in [table.md](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Table/examples/Table.md) add `table-cell-IsEmpty=“true”`

Reply:
Then in [table.md](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Table/examples/Table.md), anywhere you see `{{#> table-td}}{{/table-td}}` (lines 257, 408, 409, 441, 442, 602, 603, 834, 835, 1150, 1151, 1180, 1181, 1187) add `table-td--IsEmpty="true"`. 
lines: 996 & 997 (should be `table-td table-td--IsEmpty="true"` instead of `table-th`)

Reply:
This was really helpful feedback thanks @mattnolting